cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -1438,6 +1444,7 @@ spec:
                   type: boolean
             placement: {}
             resources: {}
+            security: {}
```

Comment:
Do we need to declare this in the v1beta1 schema? I was hoping we could leave the v1beta1 schema untouched. Or was this line the whole reason the integration tests were failing on 1.15?

Reply:
I believe yes, I was looking into and thought this could be the reason and after adding CI is good (now also,CI is falling but those are known errors IIRC). I'll do run test locally to see.